File: Veterinario/VeterinarioBlueJ/Encargado.java
public class Encargado extends Persona {
    private String direccion;

    public Encargado(String id, String nombre, String telefono, String direccion) {
        super(id, nombre, telefono);
        this.direccion = direccion;
    }

    public String getDireccion() { return direccion; }
    public void setDireccion(String direccion) { this.direccion = direccion; }
}

File: Veterinario/ConsultorioVeterinario-V2/ConsultorioVeterinario/Logica/Encargado.cs
namespace ConsultorioVeterinario.Logica { 

    public class Encargado : Persona
{
    public string Direccion { get; set; }

    public Encargado(string id, string nombre, string telefono, string direccion)
        : base(id, nombre, telefono)
    {
        Direccion = direccion;
    }
}
}